play/apps/tournament/urls.py

```diff
@@ -28,7 +28,7 @@
         GET=tournament_views.tournament_bracket.new,
         POST=tournament_views.tournament_bracket.new,
     )),
-    url(r'^tournament/bracket/(?P<id>\w+)/edit/$', method_dispatch(
+    url(r'^tournament/bracket/(?P<tournament_id>\w+)/edit/$', method_dispatch(
```

Comment:
Shouldn’t this be `bracket_id`?

Reply:
haha yeah, the code still works, I'll fix it though